e2e/_suites/ingest-manager/services.go

```diff
@@ -0,0 +1,200 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/elastic/e2e-testing/e2e"
+	log "github.com/sirupsen/logrus"
+)
+
+// ElasticAgentInstaller represents how to install an agent, depending of the box type
+type ElasticAgentInstaller struct {
+	artifactArch      string // architecture of the artifact
+	artifactExtension string // extension of the artifact
+	artifactName      string // name of the artifact
+	artifactOS        string // OS of the artifact
+	artifactVersion   string // version of the artifact
+	image             string // docker image
+	InstallCmds       []string
+	name              string // the name for the binary
+	path              string // the local path where the agent for the binary is located
+	profile           string // parent docker-compose file
+	PostInstallFn     func() error
+	service           string // name of the service
```

Comment:
service: comes from the docker-compose file, the name of the service in the YAML file